### PR TITLE
LPA: Check status return from sirius irrespective of what's status is stor…

### PR DESCRIPTION
**Pull request by SeemaMenon**
…ed in LPA db

## Purpose
https://opgtransform.atlassian.net/browse/LPA-3740

Fixes LPA-3740

## Approach

Removed condition and send requests to Sirius to check status for all LPA's irrespective of their current status in LPA database. Based on the response received from gateway if the status differs from what is in lpa db, update database.

## Learning

_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Refunds service_

## Checklist

* [X] I have performed a self-review of my own code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [X] I have added tests to prove my work
* [ ] The product team have tested these changes

### CODE REVIEW

File: service-api/module/Application/src/Controller/StatusController.php [hunk 1]
```diff
@@ -248,7 +226,7 @@ public function get($ids)
                         $rejectDate = isset($lpaDetail['rejectedDate']) ? $lpaDetail['rejectedDate'] : null;
 
                         // If it doesn't match what we already have update the database
-                        if (!is_null($lpaDetail['status']) && $lpaDetail['status']){
+                        if (!is_null($lpaDetail['status']) && $lpaDetail['status'] && $lpaDetail['status'] != $currentProcessingStatus){
```
Comment (cooperaj) [inline service-api/module/Application/src/Controller/StatusController.php:229]: it's possible to change this line to read 
`if (isset($lpaDetail['status']) && $lpaDetail['status'] !== $currentProcessingStatus) {`

(untested)

Note: always do an explicit test `!==`

File: service-api/module/Application/tests/Controller/Version2/Lpa/StatusControllerTest.php [hunk 2]
```diff
@@ -345,43 +365,89 @@ public function testGetWithSameStatus()
...
+        $this->processingStatusService->shouldReceive('getStatuses')
+            ->once()
+            ->andReturn([
+                '98765' => ['status' => 'Returned','rejectedDate' => new DateTime('2019-02-10')]
+            ]);
 
         $result = $this->statusController->get('98765');
 
-        $this->assertEquals(new Json(['98765' => ['found'=>true, 'status'=>'Returned']]), $result);
+        $this->assertEquals(new Json(['98765' => ['found' => true, 'status' => 'Returned', 'receiptDate' => null,'registrationDate' => null, 'rejectedDate' => new DateTime('2019-02-10') ]]), $result);
```
Comment (cooperaj) [inline service-api/module/Application/tests/Controller/Version2/Lpa/StatusControllerTest.php:450]: This line will be too long. Please add newlines as required (same goes for other over length lines)

**Review by cooperaj — APPROVED** [review]
LGTM

**Review by nickdavis2001 — APPROVED** [review]
lgtm

**Pull request merged**